test/testSamples/testMayaDisplayModes/testMayaDisplayModes.ma

```diff
@@ -1,19 +1,19 @@
 //Maya ASCII 2026ff01 scene
 //Name: testMayaDisplayModes.ma
-//Last modified: Wed, Mar 06, 2024 12:10:27 PM
+//Last modified: Tue, Apr 16, 2024 02:39:26 PM
```

Comment:
why the scene needs an update?

Reply:
The prior ref images were too dark. This is because our test framework resets the default light intensity to 1 (from ~3) and that code also turns off Color Management (inside [applyTestSettings()](https://github.com/Autodesk/maya-hydra/blob/dev/test/testUtils/mayaUtils.py#L144)). To use default light intensity( for brighter ref image), the test scene is opened by not calling the utility fn like so: 
testFile = mayaUtils.openTestScene(
                "testMayaDisplayModes",
                "testMayaDisplayModes.ma", useTestSettings=False)
So the test scene was now modified where CM setting is manually turned off in the file.  Note that the CM needs to be off in our test for various reasons.